src/OrchardCore.Modules/OrchardCore.ContentLocalization/GraphQL/LocalizationInputObjectType.cs

```diff
@@ -13,6 +13,7 @@ public LocalizationInputObjectType(IStringLocalizer<LocalizationInputObjectType>
             Description = S["the localization part of the content item"];
 
             AddScalarFilterFields<StringGraphType>("culture", S["the culture of the content item to filter"]);
+            Field(x => x.LocalizationSet).Description(S["the localization set for your content item."]);
```

Comment:
for consistency this should probably be `the localization set of the content item to filter`

Reply:
done